climada_petals/hazard/river_flood.py

```diff
@@ -137,22 +136,22 @@ def from_nc(cls, dph_path=None, frc_path=None, origin=False,
             if ISINatIDGrid:
 
                 dest_centroids = RiverFlood._select_exact_area(countries, reg)[0]
-                meta_centroids = copy.copy(dest_centroids)
-                meta_centroids.set_lat_lon_to_meta()
+                centroids_meta = dest_centroids.get_meta()
```

Comment:
Seems like the linter is not up-to-date with the latest version of `climada_python`? It complains that `Centroids.get_meta` does not exist, but it does in `develop`. This might be related to the changes in the Jenkins scripts.

Reply:
Right. The develop environment of climada_petals points to core package before the refactoring. Opposed to the `unit_test` stage, the `lint` stage does not change to the venv environment on top. That's why.

Reply:
If we revert changes to this setup anyway, no worries :v: